entries/t/theta.tv.json

```
@@ -0,0 +1,12 @@
+{
+  "Theta.tv": {
+    "domain": "theta.tv",
```

```suggestion
    "domain": "theta.tv",
    "img": "theta.tv.png",
```

The `img` key is required if the image is not an SVG